migrations/2_sample_contract.js

```diff
@@ -0,0 +1,12 @@
+var SampleContract = artifacts.require("./SampleContract.sol");
+const helper = require('./migration_utils.js');
+
+module.exports = async(deployer, network, accounts) => {
+  await SampleContract.deployed()
+
+  await helper.saveContractData(
+    web3,
+    [SampleContract],
+    "Singleton"
+  )
+}
```

Comment:
I'm fine with this, but after seeing truffle db. I realize I've forgotten about referencing the ABI. But I assumed that if you have the code for that deployment you can look at the ABI from that commit.

This is not a problem, but just something I'd like to note.